cli.js

```diff
@@ -24,7 +24,42 @@
 */
 
 load("./shell-config.js")
-load("./JetStreamDriver.js");
+
+const cliFlags = { __proto__: null };
+const cliArgs = [];
+if (globalThis.arguments?.length) {
+    for (const argument of globalThis.arguments)
+        if (argument.startsWith("--")) {
+            const parts = argument.split("=");
+            cliFlags[parts[0].toLowerCase()] = parts.slice(1).join("=");
+        } else
+            cliArgs.push(argument);
+}
+
+function getIntFlag(flags, flag) {
+    if (!(flag in flags))
+        return undefined;
+    const rawValue = flags[flag];
+    const value = parseInt(rawValue);
+    if (value <= 0)
+        throw new Error(`Expected positive value for ${flag}, but got ${rawValue}`);
+    return value;
+}
+
+globalThis.testIterationCount = getIntFlag(cliFlags, "--iteration-count");
+globalThis.testWorstCaseCount = getIntFlag(cliFlags, "--worst-case-count");
+
+if ("--dump-json-results" in cliFlags)
+    globalThis.dumpJSONResults = true;
```

Comment:
optional nit: `globalThis.dumpJSONResults = "--dump-json-results" in cliFlags;` for consistency with the lines above.